_layouts/job_post.html

```diff
@@ -3,70 +3,68 @@
 title_prefix: "Jobs with Compiler:"
 ---
 
-<div class="container posting">
```

Comment:
This parent `container` is no longer necessary b/c the default layout has a `container-fluid` parent already.